.github/workflows/addToProject.yml

```diff
@@ -6,6 +6,9 @@ on:
 env:
   MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
 
+permissions:
+  contents: read
```

Comment:
I'm not sure if `elastic/assign-one-project-github-action@1.2.2` requires some write access

Reply:
@AlexanderWert it looks like this github action is assigning our issues to the `OLD` apm-agents project anyway. Should it be disabled or it is still useful?

Reply:
Removed in https://github.com/elastic/apm-agent-python/pull/1972/commits/febb916fd524cc22020a11cb5b0cafd8c6038e44